cv_next/server/api/cvs.ts

```diff
@@ -75,37 +68,32 @@ export async function getPaginatedCvsByQueryFilter(
     }
     pageQuery =
     queryFilter !== undefined
-    ? query(
-        collectionRef,
-        orderBy("uploadDate", "desc"),
-        limit(queryLimit),
-        startAfter(lastDoc),
-        filterDeletedQuery,
-        queryFilter
-      )
-    : query(
-        collectionRef,
-        orderBy("uploadDate", "desc"),
-        limit(queryLimit),
-        startAfter(lastDoc),
-        filterDeletedQuery
-      );
+    ? 
+    collectionRef
+    .orderBy("uploadDate", "desc")
+    .limit(queryLimit)
+    .where(filterDeletedQuery)
+    .where(queryFilter)
+    .startAfter(lastDoc)
```

Comment:
If we're using a package that let's us use "skip", could you please refactor this so that instead of passing a `lastDoc` you'll receive a page number and skip according to page number * pre-set amount of cvs in page (can also receive this, with a default amount of let's say 10)

Reply:
Good idea. Thanks. I'll look into that.

Reply:
Yeah well I looked into that - pretty sure that StartAt\After use the doc id, like the original implementation. Offset, on the other hand, is recommended against, in multiple sources, such as this one:
https://firebeast.dev/tips/do-not-use-offset
Apparently, "offset" severely increases IO. In the meantime, I'll keep the original implementation